docs/src/components/Layout/index.tsx

```diff
@@ -138,7 +138,7 @@ export default function Page({
         <Footer />
       </main>
 
-      {hideToc ? null : (
+      {hideToc || !headings.length ? null : (
```

Comment:
This might end up being really rare, but some of our pages don't have any h2 headings, like the new placeholder-ish Usage pages for Vue, Angular, and Flutter from this PR, so it seems like we should hide this empty Table of contents.

Reply:
would if be cleaner if we do something like...? Just IMO

```
{hideToc && headings.length && (
```